phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/SetupWizardBrowser.java

```diff
@@ -0,0 +1,5 @@
+package org.phoenicis.scripts.wizard;
+
+interface SetupWizardBrowser extends Wizard {
```

Comment:
JavaDoc + the interface name sounds weird, maybe better `BrowserSetupWizard`?

Reply:
It was not used